backend/utils/config.js

```
@@ -318,10 +318,10 @@ export class ConfigService {
         };
         this.setSSOParams = (config) => {
             if (process?.env?.RTL_SSO) {
-                config.SSO.rtlSso = +process?.env?.RTL_SSO;
+                config.SSO.rtlSSO = +process?.env?.RTL_SSO;
             }
             else if (config.SSO && config.SSO.rtlSSO) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)